ports/stm32/boards/NUCLEO_F446RE/mpconfigboard.h

```diff
@@ -24,12 +24,12 @@
 #define MICROPY_HW_UART_REPL_BAUD   115200
 
 // I2C busses
-#define MICROPY_HW_I2C1_SCL (pin_B6)        // Arduino D10, pin 17 on CN10
-#define MICROPY_HW_I2C1_SDA (pin_B7)        //              pin 21 on CN7
+#define MICROPY_HW_I2C1_SCL (pin_B8)        // Arduino D15, pin 3 on CN10
+#define MICROPY_HW_I2C1_SDA (pin_B9)        //         D14, pin 5 on CN10
 #define MICROPY_HW_I2C2_SCL (pin_B10)       // Arduino D6,  pin 25 on CN10
 #define MICROPY_HW_I2C2_SDA (pin_B3)        // Arduino D3,  pin 31 on CN10
 #define MICROPY_HW_I2C3_SCL (pin_A8)        // Arduino D7,  pin 23 on CN10
-#define MICROPY_HW_I2C3_SDA (pin_C9)        //              pin  1 on CN10
+#define MICROPY_HW_I2C3_SDA (pin_B4)        // Arduino D5,  pin 27 on CN10
```

Comment:
I'm not sure this change makes things any better.  What is the reason for it?

Reply:
I am not sure either...there is nothing in the official st docs regarding I2C3 but I found this information on the NucleoF446RE Mbed page https://os.mbed.com/platforms/ST-Nucleo-F446RE/

Reply:
I2C3_SDA is also labelled as being on PC9 in that mbed documentation.  So best not to change things if they already work and there's no compelling reason to change them.